### <span style="color:green">Task 1</span>

Пусть имеется выборка $X_1,...,X_n$, где $X_i \sim U(\theta, \theta+1)$.

В качестве состоятельной и несмещённой оценки параметра $\theta$ можно взять $\overline{X}-\frac{1}{2}$. Действительно:

$$\mathbb{E}\left[\overline{X}-\frac{1}{2}\right] = \mathbb{E}X_i - \frac{1}{2} = \frac{\theta + \theta + 1}{2}-\frac{1}{2} = \theta$$

$$\mathbb{D}\left[\overline{X}-\frac{1}{2}\right] = \mathbb{D}\left[\overline{X}\right] = \frac{\mathbb{D} X_i}{n} = \frac{\left(\theta + 1 - \theta\right)^2}{12n} \xrightarrow[n \to \infty]{}0$$

Так как у данного распределения конечная дисперсия, то $\overline{X}$ является асимптотически нормальной оценкой математического ожидания, то есть:

$$\sqrt{n}\left(\overline{X} - \left(\theta +\frac{1}{2}\right)\right) \sim N\left(0, \sigma^2\left(\theta +\frac{1}{2}\right)\right)$$

Очевидно тогда, что:

$$\sqrt{n}\left(\left(\overline{X}-\frac{1}{2}\right) - \theta\right) \sim N\left(0, \sigma^2\left(\theta\right)\right)$$

### <span style="color:green">Task 2</span>

Имеем выборку $\mathbb{X} = (98, 102, 114, 86, 79, 94, 99, 101, 107, 105, 98, 111, 100, 93)$ из распределения Пуассона. Построим методом наибольшего правдоподобия оценку параметра $\lambda$ и покажем, что она является эффективной.

Сперва посчитаем количество и  сумму элементов в выборке:

In [3]:
X = (98, 102, 114, 86, 79, 94, 99, 101, 107, 105, 98, 111, 100, 93)

len(X)

14

In [4]:
sum(X)

1387

Затем выведем функцию правдоподобия:

$$\mathcal{L}(\mathbb{X}, \lambda) = \frac{\lambda^{1387}}{98!102!...93!}e^{-14\lambda}$$

Затем посчитаем информанту первого порядка и решим уравнение правдоподобия:

$$U_I(\mathbb{X}, \lambda) = \frac{\partial\ln\mathcal{L}(\mathbb{X}, \lambda)}{\partial\lambda} = \frac{\ln\left(\frac{1}{98!102!...93!}\right)+1387\ln\lambda - 14\lambda}{\partial\lambda} = \frac{1387}{\lambda} - 14 = 0 \Rightarrow \hat{\lambda} = \frac{1387}{14}$$

В итоге в качестве оценки получили выборочное среднее. Как известно, оно является эффективной оценкой параметра $\lambda$ для распределения Пуассона.

### <span style="color:orange">Task 3</span>

Докажем несмещённость исправленной выборочной дисперсии:

$$\mathbb{E}S^2 = \frac{1}{n-1}\mathbb{E}\left[\sum_{i=1}^n\left(X_i-\overline{X}\right)^2\right] = \frac{1}{n-1}\mathbb{E}\left[\sum_{i=1}^nX_i^2-2\overline{X}\sum_{i=1}^nX_i+n\overline{X}^2\right] = \frac{1}{n-1}\mathbb{E}\left[\sum_{i=1}^nX_i^2-n\overline{X}^2\right] = \frac{1}{n-1}\sum_{i=1}^n\mathbb{E}X_i^2 - \frac{n}{n-1}\mathbb{E}\overline{X}^2 = \frac{n}{n-1}\left(\mathbb{E}X^2-\frac{1}{n^2}\mathbb{E}\left(\sum_{i=1}^nX_i\right)^2\right) = \frac{n}{n-1}\left(\mathbb{E}X^2-\frac{1}{n^2}\left(n\mathbb{E}X^2+n(n-1)\left(\mathbb{E}X\right)^2\right)\right) = \mathbb{D}X$$

Докажем состоятельность исправленной выборочной дисперсии:

$$\mathbb{D}S^2 = \frac{1}{(n-1)^2}\mathbb{D}\left[\sum_{i=1}^n\left(X_i-\overline{X}\right)^2\right]$$